### DISCUSSION TITLE
data-type="ajax" and data-with / pass variable to fancybox

### DISCUSSION THREAD

**Discussion by mDesignHub** [fancyapps/ui#378] [General]
i love fancyapps and want to use it much more in my projects, but i struggle with this issues:

1) how can we set data-width and data-height in combination with data-type="ajax" ? it seems it does only work with data-type="iframe"

2) how can achieve sending a variabale to fancybox, like:
<a data-fancybox data-type="ajax" href="content.php?var=1">Open</a>

3) how can i pass a class to the opening window for individual styling the window
<a data-fancybox data-type="ajax" data-class="custom" href="content.php">Open</a>

**Comment by fancyapps** [comment]
The idea of HTML/inline content is that you set dimensions by yourself. So, for example, you simply return content like `<div style="max-width:500px">...</div>`.

You can use `load` event to access carousel slide (`$el`) and element containing the content (`$content`), you can then do whatever you need - add class name, set style directly, ...
